alpha/declcfg/write.go

```diff
@@ -426,8 +426,8 @@ func writeToEncoder(cfg DeclarativeConfig, enc encoder) error {
 		}
 
 		deprecations := deprecationsByPackage[pName]
-		sort.SliceStable(deprecations, func(i, j int) bool {
-			return deprecations[i].Name < deprecations[j].Name
+		sort.Slice(deprecations, func(i, j int) bool {
+			return deprecations[i].Package < deprecations[j].Package
```

Comment:
I realized my suggestion to change to sort by package name results in this being a no-op because `deprecations` is all of the `Deprecation` objects with the same `pName`.

Perhaps we just _don't_ sort? Because we expect this `deprecations` slice to be length 0 or 1.

Reply:
That wasn't you.  I did that 9 days ago, and you're right ... with ordinality [0,1] there's no need to sort.